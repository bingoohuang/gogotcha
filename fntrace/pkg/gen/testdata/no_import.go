package main

func A1() {
	B1()
}

func B1() {
	C1()
}

func C1() {
	D()
}

func D() {
}

func main() {
	A1()
}
