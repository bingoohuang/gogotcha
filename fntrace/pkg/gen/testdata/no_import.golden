package main

import "github.com/bingoohuang/fntrace"

func A1() {
	defer fntrace.Trace()()
	B1()
}

func B1() {
	defer fntrace.Trace()()
	C1()
}

func C1() {
	defer fntrace.Trace()()
	D()
}

func D() {
	defer fntrace.Trace()()
}

func main() {
	defer fntrace.Trace()()
	A1()
}
